modules/vector-search/pages/create-vector-search-index-ui.adoc

```diff
@@ -0,0 +1,74 @@
+= Create a Vector Search Index in Quick Mode
+:page-topic-type: guide
+:description: Use Quick Mode to create a Vector Search index in Couchbase Capella.
+
+[abstract]
+{description}
+
+Use xref:create-search-index-ui.adoc[Advanced Mode] to add additional settings and more advanced options for your Vector Search index and xref:customize-index.adoc#type-mappings[type mappings].
+
+You must create a Vector Search index before you can xref:run-vector-search-ui.adoc[run a search] that supports vector comparisons.
+
+== Prerequisites 
+
+* You have the Search Service enabled on a node in your database.
+For more information about how to change Services on your database, see xref:cloud:clusters:modify-database.adoc[].
+
+* You have a bucket with scopes and collections in your database. 
+For more information, see xref:cloud:clusters:data-service/manage-buckets.adoc[].
+
+* You have documents in a keyspace inside that bucket that contain vector embeddings.
+
+* You have logged in to the Couchbase Capella UI. 
+
+== Procedure 
+
+To create a Vector Search index with Quick Mode in Capella:
+
+. On the *Databases* page, select the database where you want to create a Search index.
+. Go to menu:Data Tools[Search].
+. Click btn:[Create Search Index].
++
+By default, *Quick Mode* should be on.
+. In the *Index Name* field, enter a name for the Vector Search index. 
++
+[NOTE]
+====
+Your index name must start with an alphabetic character (a-z or A-Z). It can only contain alphanumeric characters (a-z, A-Z, or 0-9), hyphens (-), or underscores (_).
+
+For version 7.6 and later, your index name must be unique inside your selected bucket and scope. You cannot have 2 indexes with the same name inside the same bucket and scope.
+====
+. Under *Type Mappings*, in the *Bucket* list, select the bucket that contains the documents you want to include in your index. 
+. In the *Scope* list, select the scope that contains these documents. 
+. Expand the collection that contains these documents. 
+. In your document schema, select a field that contains your vector embeddings. 
++
+Vectors should be an array of floating point numbers.
+. Configure the options for your new child field as follows:
```

Comment:
I added a note to the previous step - let me know if that resolves your question?

Reply:
I guess I still don't understand if there's a new field (child or otherwise) being added or if I'm just adding configuration to something I selected.

Reply:
So, you have the document schema displayed. You select the field from the JSON schema that contains the data you want to be able to search from your index. 

That's the child field, the field you selected. 

Then, you're adding configuration onto that. 

You select the field on one side, and configure on the other.